project-base/storefront/pages/_app.tsx

```diff
@@ -13,6 +14,8 @@ import 'styles/user-text.css';
 import { logException } from 'utils/errors/logException';
 import { initDayjsLocale } from 'utils/formaters/formatDate';
 
+const framerMotionPlugins = () => import('utils/animations/framerMotionPlugins').then((res) => res.default);
+
```

Comment:
Interesting approach. Is this done based on some documentation?

Reply:
Yes: 
https://www.framer.com/motion/guide-reduce-bundle-size/